### ISSUE TITLE
Ignore .svn and .git directories per default

### ISSUE THREAD

**Issue by arm-in** [codespell-project/codespell#783]
Hello,

I would like to ignore hidden directories like .svn or .git per default

Rationale:
* Yes, ignoring directories is possible via option -S, --skip or using xargs (see #60 and #99)
* Hidden files have to be enabled via option -H, --check-hidden
* However, hidden directories like .svn and .git are checked per default. This sounds inconsequent / illogical

**Comment by luzpaz** [comment]
This makes sense to me. Though for the purists, we could add a flag to enable search `.svn` and `.git` files.

**Comment by larsoner** [comment]
I agree that if hidden files are not checked, hidden dirs should not be, either. Then `-H` should enable both.

**Comment by luzpaz** [comment]
:+1:

**Comment by akien-mga** [comment]
I came here to ask for this feature too, I was surprised when seeing this in my log right now:
```
$ codespell -w -q 3 --skip="./thirdparty,./editor/translations" -I ../godot-word-whitelist.txt
[...]
./.git/logs/HEAD:50: Readd  ==> Readd, re-add, read
./.git/logs/HEAD:600: Readd  ==> Readd, re-add, read
./.git/logs/HEAD:604: Readd  ==> Readd, re-add, read
./.git/logs/HEAD:685: Readd  ==> Readd, re-add, read
./.git/logs/HEAD:687: Readd  ==> Readd, re-add, read
./.git/logs/HEAD:688: Readd  ==> Readd, re-add, read
FIXED: ./.git/logs/HEAD
./.git/logs/refs/heads/master:279: Readd  ==> Readd, re-add, read
```

**Comment by mandarvaze** [comment]
`.venv` folder is also being checked, which does not make sense. IMO, it should also be skipped, and needs to be explicitly checked via `-H` option

**Comment by dseight** [comment]
As a workaround it's possible to add shell alias like this one:
```bash
alias codespell="codespell --skip='.git,.svn,.venv'"
```
#1351 should help with extending per-project ignore list

**Comment by arm-in** [comment]
See also 166147f5765ce35d77cbf1e20b495ce640314c16
